scripts/crypto_materials_generator.js

```diff
@@ -9,8 +9,9 @@
  * GitHub history for details.
  */
 
-import { CryptoCli } from './cli';
+require('../src/setup_node_env');
 
-const args = require('yargs').argv;
+var args = require('yargs').argv;
```

Comment:
When declaring variables and constants, use the [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) and [const](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) keywords, not [var](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var).

https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/JavaScript#declaring_variables

Reply:
/Volumes/workplace/OSDCM/scripts/crypto_materials_generator.js
        14:1  error  Using 'VariableDeclaration[kind="const"]' is not allowed            no-restricted-syntax
        15:1  error  Using 'VariableDeclaration[kind="let"]' is not allowed              no-restricted-syntax
        15:5  error  Using 'VariableDeclarator[id.type="ObjectPattern"]' is not allowed  no-restricted-syntax

lint failed